apps/examples/i2schar/i2schar_receiver.c

```diff
@@ -152,6 +155,33 @@ pthread_addr_t i2schar_receiver(pthread_addr_t arg)
 
 	sleep(2);
 
+	for (i = 0; i < CONFIG_EXAMPLES_I2SCHAR_TXBUFFERS; i++) {
```

Comment:
Why do you want to add this transmit part in receiver file?

Reply:
It playbacks recorded audio in RX thread.